infra/development/docker-compose.yml

```
@@ -0,0 +1,18 @@
+    ports:
+      - "${DOCKER_ONEPIECE_API_PORT}:8080"
+    env_file:
+      - .env
+    environment:
+      - SPRING_PROFILES_ACTIVE=dev
+    networks:
+      - onepiece-network
+
+networks:
+  onepiece-network:
+    driver: bridge
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)